src/screens/pos-actions/pos-basic-action/pos-basic-action.tsx

```diff
@@ -131,11 +131,12 @@ export class PosBasicActionComponent extends React.Component<
 
         const tokenConfig = getTokenConfig(this.props.blockchain, this.props.token.symbol);
 
+        const balance = new BigNumber(this.props.validators[0].amountDelegated.active).plus(
```

Comment:
Is this safe for all blockchains?

Reply:
yes, pending is mandatory field added when i implemented Celo, and for near and zil its returned 0